src/transformers/__init__.py

```diff
@@ -3750,6 +3751,7 @@
             PhrasalConstraint,
         )
         from .generation_beam_search import BeamScorer, BeamSearchScorer, ConstrainedBeamSearchScorer
+        from .generation_contrastive_search import ContrastiveDecodingOneStepFast
```

Comment:
I would maybe not put this in the public init as it could be prone to change in the future and I don't think most people will use it outside of `generate` no?